core/src/main/java/org/infinispan/interceptors/impl/CacheWriterInterceptor.java

```diff
@@ -243,17 +247,16 @@ public Object visitWriteOnlyKeyCommand(InvocationContext ctx, WriteOnlyKeyComman
                   storeEntry(ctx, key, command);
                }
             }
-            break;
```

Comment:
If only the other case did something bad and fail a test, then I would have noticed it ;)